REFERENCE.md

```diff
@@ -2352,7 +2352,48 @@ Struct[{
     Optional['IOSchedulingPriority']      => Variant[String[0,0],Integer[0,7]],
     Optional['SyslogIdentifier']          => String,
     Optional['LogLevelMax']               => Enum['emerg','alert','crit','err','warning','notice','info','debug'],
+    Optional['LimitCPU']                  => Pattern['^\d+(s|m|h|d|w|M|y)?(:\d+(s|m|h|d|w|M|y)?)?$'],
```

Comment:
Just need some type checking in `spec/type_units/systemd_unit_service_spec.rb` copying some from the existing type makes sense.

Once this is merged (and released) am happy to do deprecating merge of `systemd::servicelimits` it can sit there till next major release.

Reply:
:+1:  took tests from `systemd_servicelimits_spec.rb`